src/main.rs

```diff
@@ -168,6 +171,11 @@ fn handle_shp(file_path: &str, simplification: f64, is_area: bool) -> Vec<GridGe
         .collect()
 }
 
+fn handle_polyline(input_str: String, precision: u32) -> Vec<GridGeom<f64>> {
```

Comment:
We'll still want to pass the `simplification` argument and use `GridGeom::vec_from_geom` here, but since it can't return areas we can exclude the `is_area` parameter in this function and just pass `false` to `vec_from_geom`